dotnet-test-samples/apigw-lambda-ddb/src/ServerlessTestApi.Infrastructure/DataAccess/DynamoDbProducts.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using ServerlessTestApi.Core.DataAccess;
+using ServerlessTestApi.Core.Models;
+
+namespace ServerlessTestApi.Infrastructure.DataAccess
+{
+    public class DynamoDbProducts : ProductsDAO
+    {
+        private static string PRODUCT_TABLE_NAME = Environment.GetEnvironmentVariable("PRODUCT_TABLE_NAME");
```

Comment:
This value is not marked as null and is not checked later, leading to a potential exception when calling the `GetItemRequest` constructor on line 29. Consider throwing immediately when this is null, as there will be no chance of success at that point.
```suggestion
        private static string PRODUCT_TABLE_NAME = Environment.GetEnvironmentVariable("PRODUCT_TABLE_NAME") ?? throw new ...Exception("...");
```